lib/pdf_fill/filler.rb

```diff
@@ -78,8 +126,15 @@ def process_form(form_id, form_data, form_class, file_name_extension, fill_optio
         form_data: form_class.new(form_data).merge_fields(fill_options),
         pdftk_keys: form_class::KEY
       )
+
+      template_path = if form_class.const_defined?(:TEMPLATE)
+                        form_class::TEMPLATE
+                      else
+                        "lib/pdf_fill/forms/pdfs/#{form_id}.pdf"
+                      end
+
```

Comment:
can clean this up a bit:
```suggestion
      template_path = "lib/pdf_fill/forms/pdfs/#{form_id}.pdf"
      template_path =  form_class::TEMPLATE if form_class.const_defined?(:TEMPLATE)
```